pages/dkp/kommander/2.0/workspaces/workspace-platform-services/index.md

```diff
@@ -28,32 +28,30 @@ To use the CLI to deploy or uninstall applications, see [Application Deployment]
 <p class="message--important"><strong>IMPORTANT: </strong>There may be dependencies between the applications, which are listed <a href="./platform-service-dependencies/">here</a>. Review them carefully prior to customizing to ensure that the applications are deployed successfully.</p>
 
 ## Workspace platform applications
-
-| NAME                          | APP ID                | Deployed by default |
-| ----------------------------- | --------------------- | ------------------- |
-| cert-manager-0.2.7            | cert-manager          | True                |
-| external-dns-2.20.5           | external-dns          | False               |
-| fluent-bit-0.16.2             | fluent-bit            | False               |
-| gatekeeper-0.6.8              | gatekeeper            | False               |
-| grafana-logging-6.13.9        | grafana-logging       | False               |
-| grafana-loki-0.33.1           | grafana-loki          | False               |
-| istio-1.9.1                   | istio                 | False               |
-| jaeger-2.21.0                 | jaeger                | False               |
-| kiali-1.29.0                  | kiali                 | False               |
-| kube-oidc-proxy-0.2.5         | kube-oidc-proxy       | False               |
-| kube-prometheus-stack-16.13.1 | kube-prometheus-stack | True                |
-| kubecost-0.17.1               | kubecost              | True                |
-| kubernetes-dashboard-4.0.3    | kubernetes-dashboard  | True                |
-| logging-operator-3.13.0       | logging-operator      | False               |
-| metallb-0.12.2                | metallb               | False               |
-| minio-operator-4.1.7          | minio-operator        | False               |
-| nvidia-0.4.2                  | nvidia                | False               |
-| prometheus-adapter-2.11.1     | prometheus-adapter    | True                |
-| reloader-0.0.99               | reloader              | True                |
-| traefik-10.3.0                | traefik               | True                |
-| traefik-forward-auth-0.3.2    | traefik-forward-auth  | True                |
-| velero-3.1.3                  | velero                | False               |
-
+| Common Name          | Application Version           | APP ID                | Deployed by default |
```

Comment:
this is updating a 2.0 page, we probably want these on 2.1 and 2.2 as well?

Reply:
done in https://github.com/mesosphere/dcos-docs-site/pull/4390/commits/867e27fa84a097576d0895fd9d84014881a946fc